### ISSUE TITLE
macOS 10.14 — Upload to MSP430FR5994 Fails

### ISSUE THREAD

**Issue by rei-vilo** [energia/msp430-lg-core#69]
Trying to upload the `Blink` example to the MSP430FR5994 LaunchPad fails with the following error message:

> DSLite version 7.2.0.2092
> Configuring Debugger (may take a few minutes on first launch)...
> 	Initializing Register Database...
> 	Initializing: MSP430
> error: MSP430: Error initializing emulator: No USB FET was found
> Failed: MSP430: Error initializing emulator: No USB FET was found
> 
> the selected serial port 
>  does not exist or your board is not connected

Trying with CCS Cloud also fails with an error message about _El Capitan_ on macOS 10.12 when macOS 10.14 _Mojave_ has just been released.

<img width="509" alt="capture 2018-10-03 a 15 43 35" src="https://user-images.githubusercontent.com/951776/46415308-4c24d600-c725-11e8-884c-78432fca0015.png">

What are the possible alternatives?

**Comment by rei-vilo** [comment]
Same with updated board package **MSP430** `1.0.4` with **DSLite** `8.2.0`.

```
DSLite version 8.2.0.1400
Configuring Debugger (may take a few minutes on first launch)...
	Initializing Register Database...
	Initializing: MSP430
error: MSP430: Error initializing emulator: No USB FET was found
Failed: MSP430: Error initializing emulator: No USB FET was found

the selected serial port 
 does not exist or your board is not connected
```

**Comment by rei-vilo** [comment]
See https://e2e.ti.com/support/microcontrollers/msp430/f/166/t/734145

**Comment by rei-vilo** [comment]
However, there are two `tty` ports:

```
$ ls /dev/tty.*
/dev/tty.usbmodem14201	/dev/tty.usbmodem14203
```

The exact same configuration works fine on Windows.

**Comment by rei-vilo** [comment]
No more issues with DSLite `9.3.0.1863` included in [release 1.0.7](https://github.com/energia/msp430-lg-core/releases/tag/1.0.7) to upload to the MSP430FR5994 LaunchPad.
